src/main/java/de/ellpeck/actuallyadditions/mod/blocks/BlockLaserRelay.java

```diff
@@ -259,10 +260,10 @@ public void displayHud(Minecraft minecraft, EntityPlayer player, ItemStack stack
 
                         String expl;
                         if(compass){
-                            expl = relay.getCompassDisplayString();
+                            expl = relay.getCompassDisplayString().replaceAll("\\\\n", "\n");
                         }
                         else{
-                            expl = TextFormatting.GRAY.toString()+TextFormatting.ITALIC+"Hold a Compass to modify!";
+                            expl = TextFormatting.GRAY.toString()+TextFormatting.ITALIC+StringUtil.localize("info."+ModUtil.MOD_ID+".laserRelay.mode.noCompasss").replaceAll("\\\\n", "\n");
```

Comment:
Why would the newline get so messed up that you have to replace \\\\\\\\n with \n?
If that is actually the case, please try to find a better way to solve that, because it looks really terrible and hacky to me O_o
There isn't even any new lines in that string, so I understand it even less.